src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -46,4 +55,13 @@ public void periodic() {
   public void simulationPeriodic() {
     // This method will be called once per scheduler run during simulation
   }
+  public void MoveUntill(double Inches){
+    //radius 3 in
+    Inches = Inches/18.8495559215;
```

Comment:
What are the 18.8... and 9.52 constant values?  Are these taken from experimentation, or the result of some equation, or ...?